.github/workflows/ci.yml

```diff
@@ -14,3 +14,12 @@ jobs:
         uses: actions/checkout@v3
       - name: Run unit tests
         uses: ./.github/actions/run_unit_tests
+  run_astyle:
```

Comment:
I want to Dockerize this, too, eventually, just as we did for running the unit tests. However, we aren't set up with a container registry, yet, and I want this job to run and give feedback quickly. That's at odds with building the image anew on every run. So, for now, I'm not Dockerizing this one.